src/stylesheets/components/_form-button.scss

```diff
@@ -1,13 +1,17 @@
 .form-button {
   &__button--white {
+    @extend .btn;
+
     align-items: center;
-    background: $secondary-color;
-    border-radius: rem(10px);
+    width: 100%;
+    height: rem(55px);
+
     color: map-get($brand-gray, 'transparent');
-    font-size: map-get($font-sizes, 'lg');
     font-weight: map-get($font-weight, 'bolder');
-    height: rem(55px);
+    font-size: map-get($font-sizes, 'lg');
     letter-spacing: -0.41px;
-    width: 100%;
+
+    background: $secondary-color;
```

Comment:
SHouldn't this colour be accessed as a map?

Reply:
Accessed every variable colors as map. Updated in [f984a14](https://github.com/rafayet-monon/surveyor/pull/13/commits/f984a140c88b61422d03e946690c357677eab51c)